webapp/client/src/components/OptionalHint.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import styled from "styled-components";
+
+const Hint = styled.span`
+  font-size: var(--text-medium);
+  font-weight: var(--font-normal);
+`;
```

Comment:
To add spacing before the optional (it looked too tight to the label to me^^)
```suggestion
const Hint = styled.span`
  font-size: var(--text-medium);
  font-weight: var(--font-normal);
  padding-left: var(--spacing-01);
`;
```

Reply:
Yeah, this is one example where I haven't checked the styling completely because this doesn't get used on the LoginPage.